package.json

```diff
@@ -4,10 +4,14 @@
   "keywords": ["quickboard", "leaderboard", "top10", "discord.js", "discord", "bot"],
   "description": "This package is a simple and easy to create leaderboard. If you're using Discord.js, you can use this package to create a leaderboard like Top 10 of the server.",
   "main": "index.js",
+  "types": "./index.d.ts",
   "scripts": {
     "start": "node index.js"
   },
-  "author": "can#0002",
+  "typesVersions": {
```

Comment:
Added the type version, only did the latest version (Accidentally changed author but changed it back)